### DISCUSSION TITLE
Example of sites made on ASP.NET-MVC-Boilerplate: Are there?

### DISCUSSION THREAD

**Discussion by biapar** [Dotnet-Boxed/Templates#59] [Q&A]
Example of sites made on ASP.NET-MVC-Boilerplate: Are there?

**Comment by RehanSaeed** [comment]
When you create a new site using the template, it has a link to my blog. I recieve a large number of referrer links from new blogs on the internet and I've checked out quite a few to see what people are doing. There are 55,500 downloads of this template not including Git clones, it's hard to say how many sites that translates to but I would guess a few thousand maybe?

I will add a section on the main page for sites built using this template and a TODO item in the ReadMe for the template itself. Thanks for the suggestion.
